apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableByteString.cs

```diff
@@ -277,5 +277,103 @@ public int CompareTo(object obj)
         /// The BigtableByteString value to convert.
         /// </param>
         public static implicit operator ByteString(BigtableByteString value) => value.Value;
+
+        // Additional operators to prevent ambiguities when comparing BigtableByteString with ByteString
```

Comment:
Both `ByteString` and `BigtableByteString` have operator overloads. So if you try to compare one to the other, the fact that they are both implicitly convertible to each other causes an ambiguity. Actually, looking at it again, `ByteString` only has `==` and `!=`, so we really only needed those, but if the others are added in the future, it could cause a breaking change in code using this library. I guess another alternative is to make the conversion from `BigtableByteString` to `ByteString` explicit instead of implicit.

Reply:
Yes, I think the latter would be a better idea. I suspect that having two-way implicit conversions can cause all kinds of issues like that... I should have caught it before.